Assignments/Sorting/SelectionSort.playground/Pages/Table of Contents.xcplaygroundpage/Contents.swift

```diff
@@ -11,3 +11,136 @@ When you ask someone to sort a deck of cards in ascending order by hand, you wil
 ****
 [Start with the first step](@next)
 */
+//
+// Part 1
+//
+
+func selectionSort(originalArray: [Int]) -> [Int] {
```

Comment:
@diegopetrucci you can us the original parameter as a variable instead of a constant and then create a local variable, e.g. ```func selectionSort(var originalArray: [Int]) -> [Int] { ... }```. Nice to see other Italians :-)